### DISCUSSION TITLE
ChromiumWebBrowser cuts left side of the page been shown

### DISCUSSION THREAD

**Discussion by ibober** [cefsharp/CefSharp#3754] [General]
- **What version of the product are you using?**
    - Assembly CefSharp.Wpf, Version=86.0., Culture=neutral, PublicKeyToken=40c4b6fc221f4138

- **What architecture x86 or x64?**
    - x64
	
- **What version of .Net?**
    - .Net 5.0
    
- **On what operating system?**
    - Win10

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    - WPF

- **What is the expected output? What do you see instead?**
![image](https://user-images.githubusercontent.com/8260211/129223263-f557aa2e-b720-4ea6-b53b-579de51eefb1.png)

- To fix it we've tried to put this in the very beginning of the `OnStartup` method of the _App.xaml.cs_:
```c#
            Cef.EnableHighDPISupport();
            var cefSettings = new CefSettings();
            cefSettings.CefCommandLineArgs.Add("disable-gpu-compositing");
            cefSettings.CefCommandLineArgs.Add("force-device-scale-factor", "1");
            Cef.Initialize(cefSettings);
```
and added this to the _app.manifest_:
```xml
      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
      <dpiAware>true/PM</dpiAware>
```

**Comment by amaitland** [comment]
Have you tested with the minimum example? https://github.com/cefsharp/CefSharp.MinimalExample

Based on photo it doesn't look like your application is correctly DPI aware.

Converted to discussion as this is unlikely a bug.

**Reply by amaitland** [reply]
Improvements were made to DPI handling in .Net 5 in version 89 (a big part of the reason we ask thatyou only open an issue if you can reproduce the problem with the latest version which is currently 91)

https://github.com/cefsharp/CefSharp/commit/e78e18fdb0198079ddcd0b4e168f0ea654101843#diff-73e4a07a77a0a008ce67b5aa18b9337842b1765a54d925440373c9930b6f3711

Don't use force-device-scale-factor when using WPF. 

You should confirm your application and the CefSharp.BrowserSubProcess.exe have the same DPI awareness in task manager.
